src/databricks/labs/ucx/cli.py

```diff
@@ -637,5 +638,13 @@ def lint_local_code(
     linter.lint(prompts, None if path is None else Path(path))
 
 
+@ucx.command
+def export_assessment(w: WorkspaceClient, prompts: Prompts):
+    """Export the UCX assessment queries to a zip file."""
+    ctx: WorkspaceContext = WorkspaceContext(w)
+    exporter : AssessmentExporter = ctx.assessment_exporter
+    exporter.export_results(prompts)
```

Comment:
```suggestion
    ctx = WorkspaceContext(w)
    exporter = ctx.assessment_exporter
    exporter.export_results(prompts)
```
those are redundant for intellij/mypy